Final changes

ichisa opened the pull request:
Sorry for doing the pull request twice. I was just practicing. These are the final codes for these two exercises.

ichisa commented:
I would want to do a new pull request with the postgis excercise. Is that possible? Or do I have to wait until the previous pull request I did is accepted first?

Sorry for doing so many pull requests.

ichisa commented:
I have deleted them. Thank you!

mmaelicke commented on WS18/Elisa/sql_postgis_exc.Rmd:
```diff
@@ -0,0 +1,170 @@
+  st_transform(ST_GeomFromText('POINT (48.0036891 7.8539392)', 4326), 25832)
+) as distance_toherdabau 
+from stations 
+limit 4
```
This limit will limit the number of stations for all  following temporary tables.That's also the reason why no HOBO locations occur in the other statements

ichisa replied:
I have erased the limit 4 in the corrected exercise but  the  there's still no result fot the query. Actually, now HOBO stations appear but every station has a similar distance, about 6300000, which is really strange.

mmaelicke commented on WS18/Elisa/sql_postgis_exc.Rmd:
```diff
@@ -0,0 +1,170 @@
+  st_transform(ST_GeomFromText('POINT (47.9958609 7.8523349)', 4326), 25832)
+) as distance_tomunster 
+from stations 
+limit 4
```
see comment above.

mmaelicke commented on WS18/Elisa/sql_postgis_exc.Rmd:
````diff
@@ -0,0 +1,170 @@
+
+```
+```{sql connection=con}
+select * from dist_to_herd where distance_toherdabau <15000
````
This does not give any resulting rows for me.... 
Maybe because of the limit used in the fisrt place (see other comments)

mmaelicke commented on WS18/Elisa/sql_postgis_exc.Rmd:
````diff
@@ -0,0 +1,170 @@
+
+```{sql connection=con}
+create temporary table tot_area_v as
+select d.name, round((sum(st_area(vc.geometry))/1000000)) as tot_area_Km2 from vegetation_cover vc join vegetation_cover_description d on vc.description_id=d.id
````
This is not exactly what was asked for:

  1. as a remark: you can solve this task in a Select, there is no need for a temporary table.
  2. the *nice formatting* shall be in the values that means the colum might be called 'area' and the value would be something like `34.1 km2`; 
Hint: You can concatenate two strings by the `||` operator, like: `round(2,343353, 1) || ' meter'` would result in `2.3 meter`

mmaelicke commented on WS18/Elisa/sql_postgis_exc.Rmd:
````diff
@@ -0,0 +1,170 @@
+
+```
+<div class="alert alert-info">
+Answer: Small differences ammong the first two coordinates systems are due to difference in precition for the location. The bigger difference in the WGS84 is because this coordinate system is in degrees, so, the result is not in meters and can not be compared with the two other coordinate system, which are in meters. 
````
Rethink this, you are almost there: 
The precision in location is the same for all three CRS as they are all using the same geometry object. We talked about transformation errors. 
The WGS84/Pseudomercartor is btw. not in degree but in meter. The difference between the first two and the Pseudomercartor is something else. Think of the 'covered' area...

ichisa commented:
I uploaded the corrections of the posgis exercise. But I don't know how to erase the Rproj files. I don't see them in githun and neither in the folder I have in my computer.

mmaelicke commented:
Either in Rstudio or from your os you can start a shell/terminal/console, however it might be called. Browse to the git project (the data course 2018 folder) and type `git rm datacourse.RProj`. Replace the filename with your actual file name.

ichisa commented:
Thank you! I think I have removed the rproj. 
I have changed what was suggested in the comments. However, there is still something not working when I try to calculate the distances of the station to Munster or Herdabau. 

Thanks a lot again.
